recipes/gdk-pixbuf/all/conanfile.py

```diff
@@ -59,17 +56,31 @@ def configure(self):
         self.settings.rm_safe("compiler.libcxx")
         self.settings.rm_safe("compiler.cppstd")
         if self.options.shared:
-            self.options["glib"].shared = True
+            wildcard = "" if Version(conan_version) < "2.0.0" else "/*"
+            self.options[f"glib{wildcard}"].shared = True
 
     def layout(self):
         basic_layout(self, src_folder="src")
 
+    def requirements(self):
+        self.requires("glib/2.76.0", transitive_headers=True, transitive_libs=True, run=can_run(self))
```

Comment:
The point isnt to let people use it


```suggestion
        self.requires("glib/2.76.0", transitive_headers=True, run=can_run(self))
```

Reply:
`glib` is a public dependency of `gdf-pixbuf`, it doesn't make any sense to me to remove `transitive_libs=True`.